src/Thermal/RadwsEOS.cpp

```diff
@@ -47,6 +46,7 @@ Radws<ndim>::Radws(Parameters* simparams, SimUnits *units):
 template <int ndim>
 Radws<ndim>::~Radws()
 {
+  delete opacity_table;
```

Comment:
This is a good argument as to why the Radws should be creating the object: it has the responsibility to delete it, so it's clearly the owner.